client/src/pages/OfferHelp.js

```diff
@@ -202,7 +226,11 @@ const OfferHelp = withRouter((props) => {
     <WizardContainer className="wizard-container">
       <Transition in={transition} timeout={250}>
         {(status) => (
-          <StyledWizard isHashEnabled status={status} nav={<WizardNav />}>
+          <StyledWizard
+            isHashEnabled
+            status={status}
+            nav={<WizardNav gtmPrefix={GTM.offerHelp.prefix} />}
```

Comment:
Do we even need this line @jyclam ?

Reply:
Yeah, it's necessary. The WizardNav needs the prefix passed in.

Reply:
Or is these for the steps?

Reply:
Some of the Wizard elements like `WizardSubmit` are rendered directly by the parent component, so we're building the complete GTM tag directly there. For others that are rendered by `WizardNav`, the end of the tag is hardcoded, so we just need to pass in the prefix

Reply:
Sounds good.